.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: "* */12 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: -1
+          operations-per-run: 300
+          remove-stale-when-updated: true
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it hasn't had any recent activity. It will be closed in 14 days if no further activity occurs.
```

Comment:
Are we going with 14 or 7?

Reply:
Should be 14.. I think 7 is ideal, but given that we're all on the project part-time it may not be enough time.

Reply:
Updated PR description.